nvflare/private/fed/server/server_runner.py

```diff
@@ -385,8 +385,8 @@ def abort(self, fl_ctx: FLContext):
         self.log_info(fl_ctx, "asked to abort - triggered abort_signal to stop the RUN")
 
     def get_persist_state(self, fl_ctx: FLContext) -> dict:
-        return {"run_number": self.run_num, "current_wf_index": self.current_wf_index}
+        return {"run_number": str(self.run_num), "current_wf_index": self.current_wf_index}
```

Comment:
if run number is changed to string,
should we change the `__init__` method type hint from int to str?

Reply:
done